eng/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  EmulatorMsiUrl: "https://aka.ms/cosmosdb-emulator"
+  StartParameters: ''
+
+steps:
```

Comment:
We already have the template in the tools repo see https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/templates/steps/cosmos-emulator.yml. We can either reference it directly or move it under eng/common.